project/Dependencies.scala

```diff
@@ -189,6 +191,7 @@ object Dependencies {
     analyticsSdk,
     specs2,
     catsEffectSpecs2,
-    slf4jSimple % Test
+    slf4jSimple % Test,
+    eventGen
```

Comment:
For now, it is used only in the tests but it isn't exactly test dependency. It would be better to leave it like this I think. Also, it isn't something added in this PR.